ui/ui-components/components/record/SampleRecordCard.tsx

```diff
@@ -473,6 +493,9 @@ const SampleRecordCard: React.FC<SampleRecordCardProps> = ({
           A Sample Record can be used to validate your configuration is
           importing the correct data to Groups and Destinations.
         </p>
+        {!isConfigUI && (
+          <p>Run a schedule on your primary source to generate records.</p>
```

Comment:
Nitpick: I'd capitalize the Grouparoo object names:

```suggestion
          <p>Run a Schedule on your Primary Source to generate Records.</p>
```